docs/apps/deluge.md

```diff
@@ -1 +1,29 @@
-Placeholder page
+# Deluge
+
+[Deluge](http://deluge-torrent.org/) is a lightweight, Free Software, cross-platform BitTorrent client.
+
+* Full Encryption
+* WebUI
+* Plugin System
+* Much more...
+
+[![deluge](https://avatars2.githubusercontent.com/u/6733935?v=3&s=200)](http://deluge-torrent.org/)
+
+## Supported Architectures
+
+Our images support multiple architectures such as `x86-64`, `arm64` and `armhf`. We utilise the docker manifest for multi-platform awareness. More information is available from docker [here](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list) and our announcement [here](https://blog.linuxserver.io/2019/02/21/the-lsio-pipeline-project/).
+
+Simply pulling `linuxserver/deluge` should retrieve the correct image for your arch, but you can also pull specific arch images via tags.
+
+The architectures supported by this image are:
+
+| Architecture | Tag |
+| :----: | --- |
+| x86-64 | amd64-latest |
+| arm64 | arm64v8-latest |
+| armhf | arm32v7-latest |
+
+- App: [[Website](https://docs.linuxserver.io/images/docker-deluge)] [[App Source](https://github.com/linuxserver/docker-deluge)]
+- Docker Image: [[Docker Hub](https://hub.docker.com/)] [[Image Source](https://hub.docker.com/r/linuxserver/deluge/)]
+
+---
```

Comment:
The whole supported architecture section looks pasted from LSIO, and the wording included things like "our images" and I'd like to not give the impression that DS is maintaining the images.